repos/fdbt-site/src/pages/csvZoneUpload.tsx

```diff
@@ -335,6 +343,45 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
             checked: selectedLineIds.includes(service.lineId),
         };
     });
+
+    if (ticket && matchingJsonMetaData) {
+        let services: string[] = [];
+        if ('exemptedServices' in ticket && ticket.exemptedServices) {
+            services = ticket.exemptedServices.map((service) => {
+                return service.lineId;
+            });
+        }
+
+        const serviceListEdit: ServicesInfo[] = chosenDataSourceServices.map((service) => {
+            return {
+                ...service,
+                checked: services.includes(service.lineId),
+            };
+        });
+        return {
+            props: {
+                csvUploadTitle: 'Upload fare zone',
+                csvUploadHintText:
+                    'Upload a fare zone as a .csv or MS Excel file. A fare zone is made up of all the relevant NaPTAN or ATCO codes within a geographical area. Refer to the help documents section to download a help file or a template.',
+                guidanceDocDisplayName: 'Download Help File - File Type PDF - File Size 826KB',
```

Comment:
Could we set these as variables somewhere and just do 

`return { props: { csvUploadTitle, csvUploadHintText,` .... etc

This way we dont have large blocks of duplicate code.